docs/dev/guidelines/client.rst

```diff
@@ -120,7 +142,7 @@ Ignoring this will lead to inconsistent spacing between icons and text.
 We use ``prettier`` to style code automatically and ``eslint`` to find additional issues.
 You can find the corresponding commands to invoke those tools in ``package.json``.
 
-11. Preventing Memory Leaks
+12. Preventing Memory Leaks
```

Comment:
Unrelated to this PR, but is there really no better way than changing every number in the whole list? Can this not be done automatically?

Reply:
Using \# seems to be the answer:
```
* This is a bulleted list.
* It has two items, the second
  item uses two lines.

1. This is a numbered list.
2. It has two items too.

#. This is a numbered list.
#. It has two items too.
```
from https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html